### PR TITLE
Updates related to the move to automotive clusters

### PR DESCRIPTION
**Pull request by jpopelka**

### CODE REVIEW

**Review by TomasTomecek — APPROVED** [review]

**Review by lachmanfrantisek — APPROVED** [review]
Looks good to me but you might want to wait for the approval of someone more experienced with openshift.

Nevertheless, thanks for the license shortening and document fixes.

File: vars/packit/prod_template.yml [hunk 1]
```diff
@@ -9,7 +9,8 @@
 project: packit-prod
 
 # Openshift cluster url
-host: https://console.pro-eu-west-1.openshift.com
+#host: https://console.pro-eu-west-1.openshift.com
+host: https://api.auto-prod.gi0n.p1.openshiftapps.com:6443
```
Comment (TomasTomecek) [inline vars/packit/prod_template.yml:13]: <3

**Review by TomasTomecek — COMMENTED** [review]
briefly went through it

File: vars/packit/prod_template.yml (hunk 1 above)
Comment (jpopelka) [reply vars/packit/prod_template.yml:13]: sorry, is this ok or not or what? :-)

**Pull request merged**